api-docs/custom-field-intro.md

```diff
@@ -3,32 +3,58 @@ title: Custom Fields API - Introduction
 category: 628c96a84164f50225dd1f14
 ---
 
-Adding relevant information beyond standard metadata like brand and model is important to help catalogue your assets. With our Custom Fields offering, Trackunit Iris can host all the data that you need. Custom Fields will enable you to capture and organize additional metadata that holds significant contextual value, thereby facilitating more robust data categorization and retrieval processes for you and your end-customers.
+With our Custom Fields offering, Trackunit Iris can host all the data that you need. Custom Fields will enable you to capture and organize additional metadata that holds significant contextual value, thereby facilitating more robust data categorization and retrieval processes for you and your end-customers.
 
-Custom fields provide a way to define new fields in the Trackunit data model. Allowing to extend and customize Trackunit Manager. Currently we support **extending the data model of assets, accounts, groups and sites with new fields**. Apart from defining your own custom fields we also provide a number of standard fields for common metadata.
+# Additional standard fields for commom metadata
+Trackunit provides a number a number of standard fields for common metadata, that can be used by all customers to extend the available data model. These fields can exists across different domains of Trackunit's data model and currently can extend assets, accounts, groups or sites.
```

Comment:
Regarding the part about fields being available across assets, accounts, groups or sites. Right now we only have standard fields on asset. Don't know if it makes to list them out.